osd/clustertransfer_starting.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "log_type": "cluster-updates",
+    "summary": "Cluster Ownership Transfer Initiated",
+    "description": "${DATE} - Cluster Ownership Transfer is starting. We expect the cluster to be available during this time.",
```

Comment:
Ideally, this would be something like:

Cluster ownership transfer initiated from user <PREV_CLUSTER_OWNER> in Red Hat account <PREV_RH_EBS_ACCOUNT_ID> to user <NEW_CLUSTER_OWNER> in the Red Hat account <NEW_RH_EBS_ACCOUNT_ID>. This process should not have any impact on the cluster or the workloads running within the cluster.

Let me know if this is not possible. Also, do we need to add the date in the message description?

Reply:
With the potential risk of leaking PII I think I agree at this point that we should not be sending the customer this information while this is still a manual process. Once we remove the human element with the Cluster Transfer improvements epic I don't think we'll be as concerned about PII since the customer will be the one initiating the transfer and accepting the transfer, so that should all be information they, in theory, had access to already, and wouldn't be a Red Hatter's human error causing a PII leak.

With that said, I went ahead and added an internal service log that sends right after the customer notification that the transfer has begun that contains more details so this information is more easily accessible to Support and SRE if we ever need to go back and look it up.